### ISSUE TITLE
Error in Flaskblog.py

### ISSUE THREAD

**Issue by sa123123** [CoreyMSchafer/code_snippets#110]
Sir,
My code have an error in python file as:-No module named forms found...
from flask import Flask,render_template,url_for
from forms import RegistrationForm,LoginForm

App=Flask(__name__)
App.config['SECRET_KEY']=''
posts=[
    {
        'author':'Sandeep',
        'title':'About Pandemic',
        'content':'COVID-19 is spreading at a very high rate',
        'date':'May 14 2020'
    },
    {
        'author':'Joe',
        'title':'About vaccine',
        'content':'Vaccine can be founded based on some microbiology rules',
        'date':'May 15 2020'
    }

]

@App.route("/")
@App.route("/home")
def home():
    return render_template('Home.html',posts=posts)

@App.route("/about")
def about():
    return render_template('About.html',title='About')

@App.route("/register")
def register():
    form=RegistrationForm()
    return render_template('register.html',title='Register',form=form)

@App.route("/login")
def login():
    form=LoginForm()
    return render_template('login.html',title='login',form=form)

if __name__=="__main__":
    App.run(debug=True)

**Comment by joeleger** [comment]
Do you have a file in your project named forms.py that contains two class's named RegistrationForm and LoginForm? You need this. It is documented and discussed in the videos and in the code contained herein.

Best wished 
Joe

**Comment by sa123123** [comment]
yes I have forms.py that contains these two classes,therefore it gives error.

**Comment by Rajendro1** [comment]
close this issue
